core/__init__.py

```diff
@@ -7,6 +7,9 @@
 
 def process_message(message):
     msg_urls = UrlsExtractor.get_music_urls(message)
+    if not msg_urls:
+        return None
```

Comment:
why not just empty return?

Reply:
This tells that the function is indeed meant to return a value for later use, and in this case, it returns None. This value None can then be used elsewhere.

https://stackoverflow.com/questions/15300550/return-return-none-and-no-return-at-all